proj/src/main/webapp/static/signup.html

```diff
@@ -61,18 +64,24 @@ <h4>welcome</h4>
               <input type="text" id="email" name="email" class="input" required><br>
               <label for="password">password:</label><br>
               <input type="password" id="pass" name="password" class="input" required><br>
+              <label for="password">confirm password:</label><br>
+              <input type="password" id="passconfirm" name="password" class="input" required><br>
```

Comment:
Is there any logic for making sure this actually matches the password?

Reply:
working on this on another pull request but there isn't any logic here